serverless-plugin/alarms-lambda.js

```diff
@@ -22,30 +22,38 @@ module.exports = function LambdaAlarms (lambdaAlarmConfig, context) {
     for (const [funcResourceName, funcResource] of Object.entries(
       lambdaResources
     )) {
-      const errAlarm = createLambdaErrorsAlarm(
-        funcResourceName,
-        funcResource,
-        lambdaAlarmConfig.Errors
-      )
-      cfTemplate.addResource(errAlarm.resourceName, errAlarm.resource)
-      const throttlesAlarm = createLambdaThrottlesAlarm(
-        funcResourceName,
-        funcResource,
-        lambdaAlarmConfig.ThrottlesPc
-      )
+      if (lambdaAlarmConfig.Errors.enabled) {
+        const errAlarm = createLambdaErrorsAlarm(
+          funcResourceName,
+          funcResource,
+          lambdaAlarmConfig.Errors
+        )
+        cfTemplate.addResource(errAlarm.resourceName, errAlarm.resource)
+      }
 
-      cfTemplate.addResource(
-        throttlesAlarm.resourceName,
-        throttlesAlarm.resource
-      )
-      const durationAlarm = createLambdaDurationAlarm(
-        funcResourceName,
-        funcResource,
-        lambdaAlarmConfig.DurationPc
-      )
-      cfTemplate.addResource(durationAlarm.resourceName, durationAlarm.resource)
+      if (lambdaAlarmConfig.ThrottlesPc.enabled) {
+        const throttlesAlarm = createLambdaThrottlesAlarm(
+          funcResourceName,
+          funcResource,
+          lambdaAlarmConfig.ThrottlesPc
+        )
+
+        cfTemplate.addResource(
+          throttlesAlarm.resourceName,
+          throttlesAlarm.resource
+        )
+      }
 
-      if (lambdaAlarmConfig.Invocations.Threshold) {
+      if (lambdaAlarmConfig.DurationPc.enabled) {
+        const durationAlarm = createLambdaDurationAlarm(
+          funcResourceName,
+          funcResource,
+          lambdaAlarmConfig.DurationPc
+        )
+        cfTemplate.addResource(durationAlarm.resourceName, durationAlarm.resource)
+      }
+
+      if (lambdaAlarmConfig.Invocations.enabled && lambdaAlarmConfig.Invocations.Threshold) {
```

Comment:
Maybe disable this one by default and mention in the docs that this one needs to be explicitly enabled.

Error if enabled but a threshold was not specified.